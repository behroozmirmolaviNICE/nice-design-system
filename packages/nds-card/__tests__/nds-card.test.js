"use strict";

const ndsCard = require("..");

describe("@nice-digital/nds-card", () => {
	it("needs tests");
});
