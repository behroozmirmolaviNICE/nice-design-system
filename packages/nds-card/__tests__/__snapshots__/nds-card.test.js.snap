// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card should match snapshot with some default attributes 1`] = `
<article
  className="card"
>
  <header
    className="card__header"
  >
    <p
      className="card__heading"
    >
      <a
        href="/about"
      >
        Card heading text
      </a>
    </p>
  </header>
  <dl
    className="card__metadata"
  >
    <div
      className="card__metadatum"
      key="item0"
    >
      <dt
        className="visually-hidden"
      >
        Email address
      </dt>
      <dd>
        john.smith@email.com
      </dd>
    </div>
    <div
      className="card__metadatum"
      key="item1"
    >
      <dt
        className="visually-hidden"
      >
        Email address
      </dt>
      <dd>
        sarah.smith@email.com
      </dd>
    </div>
  </dl>
</article>
`;
